configs/common/xiangshan.py

```
@@ -296,6 +333,52 @@ def build_xiangshan_system(args):
 
     for cpu in test_sys.cpu:
         cpu.store_prefetch_train = not args.kmh_align
+
+    # Configure trace mode if enabled
+    if hasattr(args, 'enable_trace_mode') and args.enable_trace_mode:
+        print(f"Configuring CPUs for trace mode...")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)